modules/utxo-lib/src/bitgo/UtxoTransactionBuilder.ts

```diff
@@ -1,8 +1,6 @@
-/**
- * @prettier
- */
-import * as bitcoinjs from 'bitcoinjs-lib';
 import { Transaction, TransactionBuilder } from 'bitcoinjs-lib';
+// eslint-disable-next-line
+import * as bitcoinjs from 'bitcoinjs-lib';
```

Comment:
for unclear reasons this is counted as a duplicate import, despite the listing this as allowed https://eslint.org/docs/rules/no-duplicate-imports